shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -172,7 +174,7 @@ public void surfaceDestroyed(SurfaceHolder holder) {
         addActivityLifecycleListener(platformPlugin);
         mImm = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
         mTextInputPlugin = new TextInputPlugin(this);
```

Comment:
Should the rest of `FlutterView.java` also stop using Hungarian notation?

Reply:
Short answer: yes.  Longer answer: Currently I'm operating with the approach that all new code avoids it but I'm waiting until the rework is done before altering untouched code.  That is partially to help reviewers not get confused, and secondly to keep the commit history clean with regard to the purpose of a given commit.